app/javascript/schools/courses/components/CourseEditor.res

```diff
@@ -168,6 +168,11 @@ let courseLinks = course => {
       t("course_links.edit_curriculum"),
       "fas fa-fw fa-check-square",
     ),
+    courseLink(
+      baseUrl ++ "/clone",
+      t("course_links.clone_course"),
+      "fas fa-copy",
+    ),
```

Comment:
The clone button should have a confirmation. Note that we have a `Link` component (that uses `WindowUtils.confirm`) that should make this simpler.